reorder_python_imports.py

```diff
@@ -641,20 +671,26 @@ def _version_removals(args):
 # END GENERATED
 
 
-def _is_py3(args):  # type: (argparse.Namespace) -> bool
-    for py, _ in FUTURE_IMPORTS:
-        if py.startswith('py3') and getattr(args, '{}_plus'.format(py)):
+def _is_py3(args, minor=''):  # type: (argparse.Namespace, str) -> bool
+    version = 'py3{}'.format(minor)
+    for py, _, _ in VERSION_IMPORTS:
```

Comment:
I wonder if what I did for pyupgrade would simplify things here: https://github.com/asottile/pyupgrade/pull/201